mesa/agent.py

```diff
@@ -611,6 +611,29 @@ def do(self, method: Callable | str, *args, **kwargs) -> GroupBy:
 
         return self
 
+    def count(self) -> dict[Any, int]:
+        """Return the count of agents in each group.
+
+        Returns:
+            dict: A dictionary mapping group names to the number of agents in each group.
```

Comment:
The dict is not yet typed here.

Reply:
Do we want to enforce it, if it's obvious from type hinting and/or from the description?

Reply:
I'm not sure if RTD can be configured to show types from type hinting.

Reply:
This is quite clear right? No reason to also included in the docstring, especially for small / trivial methods.

https://mesa.readthedocs.io/en/latest/apis/agent.html#mesa.agent.GroupBy.count